12_MeshLoaders/main.cpp

```diff
@@ -88,9 +92,23 @@ class MeshLoadersApp final : public MonoWindowApplication, public BuiltinResourc
 		if (!m_renderer)
 			return logFail("Failed to create renderer!");
 
-		//
-		if (!reloadModel())
-			return false;
+#ifdef NBL_BUILD_DEBUG_DRAW
+			{
+				auto* renderpass = scRes->getRenderpass();
+				ext::debug_draw::DrawAABB::SCreationParameters params = {};
+				params.assetManager = m_assetMgr;
+				params.transfer = getTransferUpQueue();
+				params.drawMode = ext::debug_draw::DrawAABB::ADM_DRAW_BATCH;
+				params.batchPipelineLayout = ext::debug_draw::DrawAABB::createDefaultPipelineLayout(m_device.get());
```

Comment:
I'm pretty sure the push constant range for the layout can be wrong here by accident see review of the main nabla PR